Gordon360/Services/RecIM/MatchService.cs

```diff
@@ -218,19 +221,21 @@ private async Task CreateMatchTeamMapping(int teamID, int matchID)
             };
             await _context.MatchTeam.AddAsync(matchTeam);
         }
-        public async Task AddParticipantAttendance(int participantID, int matchID)
+        public async Task<MatchParticipantViewModel> AddParticipantAttendance(string username, int matchID)
         {
+            var participantID = Int32.Parse(_accountService.GetAccountByUsername(username).GordonID);
             var matchParticipant = new MatchParticipant
             {
                 ParticipantID = participantID,
                 MatchID = matchID
             };
             await _context.MatchParticipant.AddAsync(matchParticipant);
             await _context.SaveChangesAsync();
+            return matchParticipant;
         }
-        public async Task UpdateTeamStats(MatchTeamPatchViewModel vm)
+        public async Task<MatchTeamViewModel> UpdateTeamStats(int matchID, MatchStatsPatchViewModel vm)
         {
-            var teamstats = _context.MatchTeam.FirstOrDefault(mt => mt.MatchID == vm.MatchID && mt.TeamID == vm.TeamID);
+            var teamstats = _context.MatchTeam.FirstOrDefault(mt => mt.MatchID == matchID && mt.TeamID == vm.TeamID);
```

Comment:
Camel-case variable names:
```suggestion
            var teamStats = _context.MatchTeam.FirstOrDefault(mt => mt.MatchID == matchID && mt.TeamID == vm.TeamID);
```